draft-ietf-add-ddr.md

```diff
@@ -172,9 +172,10 @@ protocols. For example, if the Unencrypted DNS Resolver returns three SVCB recor
 for DoH, one for DoT, and one for a yet-to-exist protocol, a client which only supports
 DoH and DoT should be able to use those records while safely ignoring the third record.
 
-To avoid name lookup deadlock, Designated Resolvers SHOULD follow the guidance
-in Section 10 of {{?RFC8484}} regarding the avoidance of DNS-based references
-that block the completion of the TLS handshake.
+To avoid name lookup deadlock, clients that use Designated Resolvers need to ensure
```

Comment:
Adding client considerations here is fine, but removing the server considerations is not. The referenced section of RFC8484 talks about server responsibilities as well, and we should preserve the original spirit there even if we do not duplicate the verbiage:

>To mitigate the possibility of deadlock, the authentication
   given DoH servers SHOULD NOT rely on DNS-based references to external
   resources in the TLS handshake.

I propose appending this following sentence to the end of the paragraph as written in this PR: "Designated Resolvers need to ensure this deadlock is avoidable as described in {{Section 10 of !RFC8484}}."

Reply:
@mstojens can you add a specific suggestion to the text that we can commit? I'm not sure exactly where you were thinking of adding the text.

Reply:
Applied